### ISSUE TITLE
February bug on Safari (Brazil)

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2711]
Originally reported on Google Code with ID 2446
```
I found a bug on Safari (iPad, iPhone and OS X) that when we select agendaWeek view
in February month of any year and then click on next or previous and then in today
button, it doesn't show the last day of the week and doesn't respond to next anymore,
to move to the last week of the month.
It can be reproduced with the example in the Intro page of http://fullcalendar.io,
so there's no need to make a JSBin or JSFiddle.

```

Reported by `cristalembsocial` on 2015-02-18 14:01:01

Imported with **2** stars.

**Comment by demartim** [comment]
The same is happening in October 16, when clicking the next button it doesn't work. I found this in Firefox.

**Comment by demartim** [comment]
In any language, changing timezone to UTC make it work. In Local ore none it still doesnt work.

**Comment by fonini** [comment]
This bug is related to Daylight Saving Time. Here in Brazil (America/Sao_Paulo), it goes from 2015-10-18 to 2016-02-21.

**Comment by arshaw** [comment]
this has likely been fixed in a beta release. please try it out!
https://github.com/fullcalendar/fullcalendar/releases/tag/v2.5.0-beta2

**Comment by arshaw** [comment]
to verify the fix, you can demo the beta release here: http://fullcalendar.io/js/fullcalendar-2.5.0-beta2/demos/agenda-views.html

**Comment by pakerwreah** [comment]
Yeah it's fixed! :+1:

**Comment by arshaw** [comment]
fixed in [v2.5.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v2.5.0)
